src/deadline/houdini_submitter/otls/deadline_cloud.hda/Driver_1deadline__cloud/DialogScript

```diff
@@ -616,6 +621,7 @@
                 default { "5" }
                 range   { 0 10 }
                 parmtag { "script_callback_language" "python" }
+                help    "The maximum number of retries for each task"
```

Comment:
Some help text has periods, and some doesn't. I think we'll need to align with the docs team about if they should have it or not.